ci/build_docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --output requirements \
+  --file_key docs \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee requirements.txt
+
+rapids-mamba-retry create -n test
+conda activate test
+pip install -r requirements.txt
```

Comment:
```suggestion
rapids-mamba-retry env create --force -f env.yaml -n docs
conda activate docs
```

These changes align with the `dependencies.yaml` suggestion.